src/openrct2/object/DefaultObjects.cpp

```diff
@@ -153,6 +153,9 @@ const std::string_view DefaultSelectedObjects[] = {
     "rct2.peep_animations.handyman",
     "rct2.peep_animations.mechanic",
     "rct2.peep_animations.security",
+
+    // Climate object
+    "rct2.climate.cool_and_wet",
```

Comment:
By default, RCT2 and OpenRCT2 select "warm" in the scenario editor. I think we should keep that.

Reply:
Ah, I was under the impression it was this one. No preference either way, so I'll change it.